tests/testthat/_snaps/emissions_profile_upstream.md

```diff
@@ -166,3 +166,58 @@
       18     0
       
 
+# FIXME? at company level, `NA` in a benchmark yields the expected `value`s (#638)
```

Comment:
@AnneSchoenauer,

Note that these three different benchmarks with similar inputs behave differently. Is this a bug? What do you actually expect in each case?